hw/top_earlgrey/lint/top_earlgrey_lint_cfgs.hjson

```diff
@@ -301,20 +301,38 @@
              {    name: chip_earlgrey_asic
                   fusesoc_core: lowrisc:systems:chip_earlgrey_asic
                   import_cfgs: ["{proj_root}/hw/lint/tools/dvsim/common_lint_cfg.hjson"]
-                  rel_path: "hw/chip_earlgrey_asic/lint/{tool}"
+                  rel_path: "hw/top_earlgrey/chiplevel/chip_earlgrey_asic/lint/{tool}"
                   overrides: [
                     {
                       name: design_level
                       value: "top"
                     }
                   ]
              },
-             // this currently causes compilation issues in AscentLint
-             //{    name: chip_earlgrey_verilator
-             //     fusesoc_core: lowrisc:systems:chip_earlgrey_verilator
-             //     import_cfgs: ["{proj_root}/hw/lint/tools/dvsim/common_lint_cfg.hjson"]
-             //     rel_path: "hw/chip_earlgrey_verilator/lint/{tool}"
-             //},
+             # TODO: this is not entirely lint clean yet. Uncomment once it is clean.
+             # {    name: chip_earlgrey_verilator
+             #      fusesoc_core: lowrisc:systems:chip_earlgrey_verilator
+             #      import_cfgs: ["{proj_root}/hw/lint/tools/dvsim/common_lint_cfg.hjson"]
+             #      rel_path: "hw/top_earlgrey/chiplevel/chip_earlgrey_verilator/lint/{tool}"
+             #      overrides: [
+             #        {
+             #          name: design_level
+             #          value: "top"
+             #        }
+             #      ]
+             # },
+             # TODO: this is not entirely lint clean yet. Uncomment once it is clean.
+             # {    name: chip_earlgrey_cw310
```

Comment:
@tjaychen we can activate this once it has been cleaned up. until then, we can leave it in the separate `cfg` file.